docs/source/templates.rst

```diff
@@ -53,6 +53,28 @@ The first line again indicates that this template extends an existing template b
 The second and last line indicate that the enclosed lines are to be placed in the *body* block of the base template.
 The third line is the actual command that we want to add and the fourth line ensures that the code provided by the base template within the body block is still added.
 
+.. note::
```

Comment:
A better place for this note is at the end of the ["Defining new environments section"](https://github.com/glotzerlab/signac-docs/blob/main/docs/source/environments.rst#defining-new-environments). That section also already provides an example of a new environment class.